python-template/{{cookiecutter.placeholder_repo_name}}/.gitignore

```diff
@@ -0,0 +1,78 @@
+*.py[cod]
+__pycache__
+.pytest_cache
+
+# C extensions
+*.so
+
+# Packages
+*.egg
+*.egg-info
+dist
```

Comment:
It would be great if we could anchor these where possible, e.g. `/dist`, so that only files in the root get ignored. Otherwise they might interfere with package names, etc.

Reply:
(I don't know which of these are likely to apply to subprojects like we have in edx-platform, but I think for special cases like that, we should expect repos to have their own additions anyhow.

Also, not sure which of these *are* appropriate to anchor to the root.)

Reply:
+1. I think the path specificity will help alleviate surprises down the road.

In case you weren't aware, you can also put `.gitignore` files in subdirectories, and they will act relative to that subdirectory.

Reply:
And importantly, anchored paths are relative to their directory!

Reply:
I agree with you guys and have made some things anchored. I didn't do it for all, cause some  paths look like they should be unanchored.